setup.py

```diff
@@ -28,12 +29,15 @@ def build_envd_if_not_found():
         logging.info("envd not found. Build from scratch")
         try:
```

Comment:
I would prefer to use https://docs.python.org/3/library/os.path.html#os.path.isfile